### ISSUE TITLE
make a favicon

### ISSUE THREAD

**Issue by elisementor** [laura-313/f1-3-c2p1-colmar-academy#2]
see "managing assets" article to learn how to make a favicon
put favicon itself in images folder
link in head of index.html

it seems like you had a link at one point but commented it out? the favicon doesn't come with the project you have to make it

https://github.com/laura-313/f1-3-c2p1-colmar-academy/blob/master/colmar-academy-final/index.html#L8

**Comment by laura-313** [comment]
yes. I had made one at one point, but was not able to get it right so i took it out. I will try again !!
